App/PHP_HTML/deleteAnswer/delete_answer_action.php

```diff
@@ -0,0 +1,14 @@
+<?php include('../sql_connector.php'); ?>
+
+// PURPOSE: runs query that deletes an answer
+
+<?php
+    $DELETED = 1;
+    $answerId = $_POST['answerId'];
+    $qus_id = $_POST['questionId'];
+
+    $query = "UPDATE answers SET answer_deleted = $DELETED WHERE answers_id = '$answerId'";
```

Comment:
Why are you adding a variable in the table "Deleted"
Couldn't you just remove it from the table in the first place?

$query = "DELETE FROM answers WHERE answers_id = '$answerId'";